documentation/htdocs/config.txt

```diff
@@ -1,4 +1,4 @@
-# Uncrustify_d-0.75.0-245-6f360ab1
+# Uncrustify-0.75.1-dev
```

Comment:
IMO, this should be 0.76.0-dev, since it is the development stage of the next release (0.76.0).

Reply:
After having tested PR #3801, I now see why the numbers are 0.75.1-dev.
I think we can discuss about updating to a newer number in a separate PR :-)